public/app/features/plugins/extensions/utils.tsx

```diff
@@ -58,7 +60,17 @@ type ModalWrapperProps = {
   onDismiss: () => void;
 };
 
-export const wrapWithPluginContext = <T,>(pluginId: string, Component: React.ComponentType<T>, log: ExtensionsLog) => {
+export const wrapWithPluginContext = <T,>({
+  pluginId,
+  extensionTitle,
+  Component,
+  log,
+}: {
+  pluginId: string;
+  extensionTitle: string;
```

Comment:
Looking at this again, do we really need the `extensionTitle` argument or would `pluginId` be enough?

Reply:
It's only used for "enriching" the error logs (and the error message on the UI), so I think we can remove it if you feel it's a bit too much. (I was using it to pinpoint which extension was the culprit.)

The ideal thing would be if the extensions would have actual IDs - something that's on the table for some time :)